modulemd/meson.build

```diff
@@ -527,3 +527,18 @@ if with_docs
         install : true,
     )
 endif
+
+if help2man.found()
```

Comment:
You don't need this `if` block. If `help2man` is a `disabler()` object, meson will automatically skip any target that uses it as a dependency.